src/KnowbaseItem.php

```diff
@@ -2206,6 +2206,7 @@ public function getAnswer()
     protected function getShowVisibilityDropdownParams()
     {
         $params = parent::getShowVisibilityDropdownParams();
+        $params['right'] = ($this->getField('is_faq') ? 'faq' : 'knowbase');
```

Comment:
`$params['right']` was deleted because it seemed to be ignored if `$params['allusers'] = 1`